build/Generator/ApiGenerator.php

```diff
@@ -61,6 +61,13 @@ public function generateOperation(ServiceDefinition $definition, string $service
         $namespace->addUse($inputClass);
         $classes = $namespace->getClasses();
         $class = $classes[\array_key_first($classes)];
+        if (null !== $version = $definition->getSignatureVersion()) {
```

Comment:
We can also check if the `$class` already have this function.

Reply:
The class must have this function (because method is abstract). But here, I want to always update, and make sure that we use the recommended version (in case AWS change it)

Reply:
Are you sure `$class->hasMethod('getSignatureVersion')` always will return true? Im actually hesitant.

Reply:
if not, the class is not loadable : 
`PHP Fatal error:  Class AsyncAws\Sqs\SqsClient contains 1 abstract method and must therefore be declared abstract or implement the remaining methods`

But you're right. I'll add a test

Reply:
Thank you